webapp/client/src/components/FormFieldDropDown.js

```diff
@@ -0,0 +1,101 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import FormFieldScaffolding from "./FormFieldScaffolding";
+import FieldLabelForSeparatedFields from "./FieldLabelForSeparatedFields";
+import selectIcon from "../assets/icons/select_icon.svg";
+
+const DropDown = styled.div`
+  & label {
+    display: block;
+    width: 30px;
+    height: 30px;
+    cursor: pointer;
+    background: white;
+    position: absolute;
+  }
+
+  & .steuerlotse-select {
+    border: 2px solid var(--border-color);
+    border-radius: 0;
+    background-image: url(${selectIcon});
+    background-repeat: no-repeat;
+    background-size: 0.75rem;
+    min-height: 55px;
+  }
+
+  & .steuerlotse-select:focus {
+    border: 2px solid var(--focus-border-color);
+    box-shadow: 0 0 0 2px var(--focus-color);
+  }
+`;
+
+function FormFieldDropDown({
+  fieldName,
+  fieldId,
+  values,
+  defaultValue,
+  preselectedValue,
+  required,
+  autofocus,
+  label,
+  details,
+  errors,
+  onChangeHandler,
+}) {
+  return (
+    <FormFieldScaffolding
+      {...{
+        fieldName,
+        errors,
+        cols: "6",
+      }}
+      hideLabel
+      render={() => (
+        <DropDown>
+          <FieldLabelForSeparatedFields {...{ label, fieldId, details }} />
+          <select
+            id={fieldId}
+            className="custom-select steuerlotse-select"
+            input_req_err_msg="Bundesland auswählen"
+            name={fieldId}
+            value={preselectedValue}
+            required={required}
+            autoFocus={autofocus || Boolean(errors.length)}
+            onBlur={onChangeHandler}
+            onChange={onChangeHandler}
+          >
+            {defaultValue && <option value="">{defaultValue}</option>}
```

Comment:
Why is the value of the default option blank?

Reply:
I could not find another way to have an empty value for the defaultValue. If I omit the value, the react component will have the name of the default option as the value (e.g. "Bitte auswählen"). However, we do not want that to have a correct value.

Reply:
That said, because of your comment about the custom validity, I am checking if it would make sense to make that a controlled input

Reply:
Aha, with the changed names, this makes perfect sense. I was just confused.

Why does the custom validity lead you in the direction of a controlled input?

Reply:
I thought that the problem might be that the react component does not set the value correctly and the validation might not work because of that. That was the reason that I thought we might need a controlled input for that.

Reply:
Interesting, ok. Guess we won't find out this time around :)